tests/web/test_outputs.py

```diff
@@ -42,8 +60,8 @@ def test_get_outputs_endpoint_with_invalid_public_key(client):
 
 
 def test_get_outputs_endpoint_with_invalid_unspent(client, user_pk):
-    expected = {'message': {'unspent': 'Boolean value must be "true" or "false" (lowercase)'}}
-    params = '?unspent=tru&public_key={}'.format(user_pk)
+    expected = {'message': {'spent': 'Boolean value must be "true" or "false" (lowercase)'}}
```

Comment:
I guess it's not an error, because you can name tests whatever you like, but the name of the test includes "with_invalid_unspent" but it really should be "with_invalid_spent" because the query parameter name is now `spent`.